docs/DeveloperGuide.md

```diff
@@ -901,30 +901,275 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+
+### Creating a person
+
+1. Creating a person with a duplicate already present
+
+   1. Prerequisite: A person with name `Alice Tan` and phone number `12345678` on the list. Other fields may be present with any value.
+
+   1. Test case: `create n/Alice Tan p/12345678`<br>
+      Expected: No new person created. Error details shown in the command display.
+
+   1. Test case: `create n/Alice Tan p/87654321`<br>
+      Expected: A new person is added to the list. Details of the new person shown in the command display.
+      
+   1. Other viable create commands to try: `create n/Bob Wang p/12345678`, `create n/alice tan p/12345678`, `create n/alice tan p/12345678`<br>
+      Expected: Success. Similar to previous.
+      
+### Assigning a PDF file to a person
+
+1. Assigning a PDF file to a person without PDF file assigned.
+   
+   1. Create a PDF file on your computer.
+   
+   2. Test case: `filepath INDEX f/FILEPATH` where FILEPATH is the absolute file path of the PDF file from step 1.<br>
+      Expected: PDF file assigned. Client details shown in the command display.
+      
+      1. Click on the client in the result display.
+      
+      2. Scroll down the contact information window.<br>
+         Expected: `Client Information` button in high-contrast white borders and is clickable.
+         
+      3. Click on the `Client Information` button<br>
+         Expected: The PDF file is opened.
+   
+   3. Test case: `filepath INDEX f/RELATIVE FILEPATH` where RELATIVE FILEPATH is the file path of the PDF relative to the directory of FABook.
+      Expected: PDF file not assigned. Error details shown in the command display.
+      
+      1. Click on the client in the result display.
+      
+      2. Scroll down the contact information window.<br>
+         Expected: `Client Information` button greyed out and unclickable.
+
+   4. Other incorrect inputs to try: Incorrect filepaths, filepaths for other file types (e.g. .doc)<br>
+      Expected: PDF file not assigned. Error details shown in the command display. See previous.
+   
+   5. See `Opening PDF file` below for more related testing.
```

Comment:
do you mean to add a link to something below?

Reply:
good idea